### PR TITLE
Added weather.IPMA

### PR DESCRIPTION
**Pull request by dgomes**
**Description:**

Adds a new weather platform with information provided by the Portuguese Weather service (IPMA)

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#14716

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/weather.ipma.markdown [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+title: "IPMA"
+description: "Instructions on how to integrate Instituto Português do Mar e Atmosfera weather conditions into Home Assistant."
+date: 2018-05-31 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: bom.png
+ha_category: Weather
+ha_release: 0.71
```
Comment (MartinHjelmare) [inline source/_components/weather.ipma.markdown]: 0.72

File: source/_components/weather.ipma.markdown [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+```yaml
+# Example configuration.yaml entry
+weather:
+  - platform: ipma 
+```
+
+{% configuration %}
+name:
+  description:  The name you would like to give to the weather station.
+  required: optional
```
Comment (MartinHjelmare) [inline source/_components/weather.ipma.markdown]: `required: false`

File: source/_components/weather.ipma.markdown [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  default: The name of the used station
+  type: string
+latitude:
+  description: "Latitude of the location for which you want weather information"
+  required: optional
```
Comment (MartinHjelmare) [inline source/_components/weather.ipma.markdown]: See above.

File: source/_components/weather.ipma.markdown [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  default: Home Assistant global latitude configuration 
+  type: string
+longitude:
+  description: "Longitude of the location for which you want weather information"
+  required: optional
```
Comment (MartinHjelmare) [inline source/_components/weather.ipma.markdown]: See above.

File: source/_components/weather.ipma.markdown [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+# Example configuration.yaml entry
+weather:
+  - platform: ipma 
+```
+
+{% configuration %}
+name:
+  description:  The name you would like to give to the weather station.
+  required: optional
+  default: The name of the used station
```
Comment (MartinHjelmare) [inline source/_components/weather.ipma.markdown]: This doesn't have a period at the end.

File: source/_components/weather.ipma.markdown [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  type: string
+latitude:
+  description: "Latitude of the location for which you want weather information"
```
Comment (MartinHjelmare) [inline source/_components/weather.ipma.markdown]: This sentence is surrounded with quotes, but other sentences are not. Also missing period.

File: source/_components/weather.ipma.markdown [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+  required: optional
+  default: Home Assistant global latitude configuration 
+  type: string
+longitude:
+  description: "Longitude of the location for which you want weather information"
```
Comment (MartinHjelmare) [inline source/_components/weather.ipma.markdown]: See above.

**Comment by dgomes** [comment]
Thank you @MartinHjelmare !

File: source/_components/weather.ipma.markdown [hunk 8]
```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "IPMA"
+description: "Instructions on how to integrate Instituto Português do Mar e Atmosfera weather conditions into Home Assistant."
+date: 2018-05-31 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: bom.png
```
Comment (MartinHjelmare) [inline source/_components/weather.ipma.markdown]: Is this the correct logo? Sorry I missed this in the first pass.

Comment (dgomes) [reply source/_components/weather.ipma.markdown]: Absolutely right! 

Completely forgot about the logo :(

**Review by MartinHjelmare — APPROVED** [review]
Great!

**Pull request merged**